### ISSUE TITLE
Link in project description gives 404

### ISSUE THREAD

**Issue by kitsunde** [wa0x6e/Fresque#58]
https://kamisama.github.com/Fresque/

<img width="820" alt="screenshot 2015-09-22 18 34 52" src="https://cloud.githubusercontent.com/assets/48965/10016288/66687818-6158-11e5-9785-812214dcd785.png">

**Comment by wa0x6e** [comment]
Yes, a lot of links are dead since the github username change. You didn't miss anything though, the link redirect to the README file.
